src/_gettsim_tests/test_piecewise_polynomial.py

```diff
@@ -0,0 +1,16 @@
+"""
+Test for BUG-fix #629
+https://github.com/iza-institute-of-labor-economics/gettsim/issues/629
+"""
+import numpy as np
+
+from _gettsim_tests._helpers import cached_set_up_policy_environment
```

Comment:
Ok thanks for the feedback. Since the function I changed `check_intercepts` is only used in `get_piecewise_parameters` and not in `piecewise-polynomial` itself, i will test `get_piecewise_parameter` for a more precise outcome.